shared/wallets/send-form/participants/to-field.tsx

```diff
@@ -82,30 +80,11 @@ export type ToStellarPublicKeyProps = {
 }
 
 const ToStellarPublicKey = (props: ToStellarPublicKeyProps) => {
-  const [recipientPublicKey, setRecipentPublicKey] = React.useState(props.recipientPublicKey)
```

Comment:
I don't really understand what happened here. What's the deal with the old debounced code / why is it no longer needed?

Reply:
this convoluted code was to try and rectify / debounce scanning qr codes on android. syncing the props / state doing this is really flakey and we should just fix it in another way

Reply:
Do we have a ticket for that?

Reply:
HOTPOT-1810